[bug] cant import plugin dashboards in optimized build

If the JS is built using "grunt build" instead of just "grunt" the dashboard import tab for plugins does not work.

```
boot.7bc1696e.js:37 Error: [$injector:unpr] Unknown provider: aProvider <- a
http://errors.angularjs.org/1.5.3/$injector/unpr?p0=aProvider%20%3C-%20a
    at boot.7bc1696e.js:35
    at boot.7bc1696e.js:35
    at Object.d [as get] (boot.7bc1696e.js:35)
    at boot.7bc1696e.js:35
    at d (boot.7bc1696e.js:35)
    at e (boot.7bc1696e.js:35)
    at Object.g [as invoke] (boot.7bc1696e.js:35)
    at l.instance (boot.7bc1696e.js:36)
    at o (boot.7bc1696e.js:36)
    at boot.7bc1696e.js:36
```

--- a/public/app/features/plugins/import_list/import_list.ts
+++ b/public/app/features/plugins/import_list/import_list.ts
@@ -10,6 +10,7 @@ export class DashImportListCtrl {
   plugin: any;
   datasource: any;
 
+  /** @ngInject */
   constructor($scope, private $http, private backendSrv, private $rootScope) {
     this.dashboards = [];
 

